libs/ui-watt/src/lib/components/modal/watt-modal.component.html

```diff
@@ -35,6 +35,9 @@ <h3 class="watt-modal-title">{{ title }}</h3>
   <ng-content select="watt-modal-actions"></ng-content>
 
   <div *ngIf="loading" class="watt-modal__spinner">
-    <watt-spinner></watt-spinner>
+    <div>
+      <watt-spinner></watt-spinner>
+      <p>{{ loadingMessage }} {{ loading }}</p>
```

Comment:
```suggestion
      <p>{{ loadingMessage }}</p>
```

Reply:
Does not seem to break anything. so i am removing the last loading variable